### PR TITLE
Added GWT-Chosen samples to dev-site

### PR DESCRIPTION
**Pull request by Elrhino**
This branch has already been deployed by @meriouma. I'm waiting for approval to merge the latest changes.

### CODE REVIEW

File: webapp/src/main/java/com/google/gwt/site/demo/gsss/animation/GSSSAnimationDemos.java [hunk 1]
```diff
@@ -31,6 +31,5 @@ public GSSSAnimationDemos(EventBus eventBus) {
         MIXINS.style().ensureInjected();
 
         registerDemo(1, Examples.create(MIXINS.style().example1()));
-        registerDemo(2, Examples.create(MIXINS.style().example2()));
```
Comment (meriouma) [inline webapp/src/main/java/com/google/gwt/site/demo/gsss/animation/GSSSAnimationDemos.java]: Was that intended?

File: webapp/src/main/java/com/google/gwt/site/demo/gwtchosen/sample/AdvancedCLBOptionsWidget.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+public class AdvancedCLBOptionsWidget implements IsWidget {
+    private static Binder uiBinder = GWT.create(Binder.class);
+
+    @UiField
+    SimplePanel hierChosenSingle;
+
+    private final Widget widget;
+
+    @UiTemplate("AdvancedCLBOptionsWidget.ui.xml")
+    interface Binder extends UiBinder<Widget, AdvancedCLBOptionsWidget> {
```
Comment (meriouma) [inline webapp/src/main/java/com/google/gwt/site/demo/gwtchosen/sample/AdvancedCLBOptionsWidget.java]: interface should be at top of class

File: webapp/src/main/java/com/google/gwt/site/demo/gwtchosen/sample/ChosenOptionsWidget.java [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    SelectElement mobileWidth;
+    @UiField
+    SelectElement mobileAnimation;
+    @UiField
+    SelectElement mobileSpeed;
+
+    private final Widget widget;
+
+    @UiTemplate("ChosenOptionsWidget.ui.xml")
+    interface Binder extends UiBinder<Widget, ChosenOptionsWidget> {
```
Comment (meriouma) [inline webapp/src/main/java/com/google/gwt/site/demo/gwtchosen/sample/ChosenOptionsWidget.java]: interface at top

File: webapp/src/main/java/com/google/gwt/site/demo/gwtchosen/sample/CustomFilteringWidget.java [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+                    // stop the repeating command
+                    return false;
+                }
+            }, 10);
+        }
+    }
+
+    @UiTemplate("CustomFilteringWidget.ui.xml")
+    interface Binder extends UiBinder<Widget, CustomFilteringWidget> {
```
Comment (meriouma) [inline webapp/src/main/java/com/google/gwt/site/demo/gwtchosen/sample/CustomFilteringWidget.java]: interface at top

File: webapp/src/main/java/com/google/gwt/site/demo/gwtchosen/sample/CustomFilteringWidget.java [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+import com.google.gwt.uibinder.client.UiField;
+import com.google.gwt.uibinder.client.UiTemplate;
+import com.google.gwt.user.client.ui.IsWidget;
+import com.google.gwt.user.client.ui.SimplePanel;
+import com.google.gwt.user.client.ui.Widget;
+
+import java.util.List;
+
+public class CustomFilteringWidget implements IsWidget {
+
```
Comment (meriouma) [inline webapp/src/main/java/com/google/gwt/site/demo/gwtchosen/sample/CustomFilteringWidget.java]: nl

File: webapp/src/main/java/com/google/gwt/site/demo/gwtchosen/sample/CustomFilteringWidget.java [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+    private static Binder uiBinder = GWT.create(Binder.class);
```
Comment (meriouma) [inline webapp/src/main/java/com/google/gwt/site/demo/gwtchosen/sample/CustomFilteringWidget.java]: fields after interfaces and static classes

File: webapp/src/main/java/com/google/gwt/site/demo/gwtchosen/sample/EventsWidget.java [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+import com.google.gwt.user.client.ui.Widget;
+
+public class EventsWidget implements IsWidget {
+
+    private static Binder uiBinder = GWT.create(Binder.class);
+
+    private final Widget widget;
+
+    @UiTemplate("EventsWidget.ui.xml")
+    interface Binder extends UiBinder<Widget, EventsWidget> {
```
Comment (meriouma) [inline webapp/src/main/java/com/google/gwt/site/demo/gwtchosen/sample/EventsWidget.java]: interface

File: webapp/src/main/java/com/google/gwt/site/demo/gwtchosen/sample/EventsWidget.java [hunk 8]
```diff
@@ -0,0 +1,27 @@
+package com.google.gwt.site.demo.gwtchosen.sample;
+
+import com.google.gwt.core.client.GWT;
+import com.google.gwt.uibinder.client.UiBinder;
+import com.google.gwt.uibinder.client.UiTemplate;
+import com.google.gwt.user.client.ui.IsWidget;
+import com.google.gwt.user.client.ui.Widget;
+
+public class EventsWidget implements IsWidget {
+
```
Comment (meriouma) [inline webapp/src/main/java/com/google/gwt/site/demo/gwtchosen/sample/EventsWidget.java]: nl

File: webapp/src/main/java/com/google/gwt/site/demo/gwtchosen/sample/EventsWidget.java [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+    private static Binder uiBinder = GWT.create(Binder.class);
```
Comment (meriouma) [inline webapp/src/main/java/com/google/gwt/site/demo/gwtchosen/sample/EventsWidget.java]: fields ordering

**Comment by olafleur** [comment]
LGTM

**Pull request merged**